### PR TITLE
DM-24637: Make CscCommander run commands asynchronously

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: tests/data/topic_writer.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                else:
+                    self.write_task = asyncio.create_task(self.write_arrays())
+        else:
+            self.write_task.cancel()
+
+    async def write_arrays(self):
+        i = 0
+        while True:
+            self.tel_arrays.data.int0[0] = i
+            self.tel_arrays.put()
```
Comment (tribeiro) [inline tests/data/topic_writer.py:89]: Why not use `set_put`? is this faster?

Comment (tribeiro) [reply tests/data/topic_writer.py:89]: never mind... I see you only want to set the first item...

File: tests/test_speed.py [hunk 2]
```diff
@@ -59,7 +66,7 @@ async def test_readwrite_speed(self):
         This isn't really a unit test, but running it as a test
         avoids bitrot.
         """
-        ntowrite = 500
+        ntowrite = 1000
```
Comment (tribeiro) [inline tests/test_speed.py:69]: picky comment could it be `n_to_write` instead? I read it "not write" the first time.. hehe

Comment (r-owen) [reply tests/test_speed.py:69]: I'll change it to num_to_write

**Pull request merged**